controls/def.cf

```diff
@@ -79,6 +79,22 @@ bundle common def
         if => and(not(isvariable("override_data_acl")), not(isvariable("acl"))),
         meta => { "defvar" };
 
+    !disable_always_accept_policy_server_acl::
+      "acl_derived" -> { "ENT-10951" }
+        slist => {
+                   "@(def.acl)",
+                   "$(sys.policy_hub)"
```

Comment:
Why are you using the `sys.policy_hub` variable here, but `sys.policy_server` in other places?

Reply:
Oh, just a bug. This pr should have been a draft.

Reply:
fixed.

Reply:
@nickanderson this one is still here ^